wandb/board/ui/src/util/plotHelpers.js

```diff
@@ -65,6 +65,53 @@ export function friendlyMetricDefaults(metricNames) {
     .slice(0, 4);
 }
 
+export function defaultRunCharts(metricNames) {
+  const charts = [],
+    big = {},
+    small = [];
+  let y = 0,
+    x = 0,
+    width = 12;
+  metricNames.forEach(name => {
+    if (name.match(/loss/)) {
+      big.loss = big.loss || [];
+      big.loss.push(name);
+    } else if (name.match(/accuracy|acc$/)) {
+      big.accuracy = big.accuracy || [];
+      big.accuracy.push(name);
+    } else if (['examples', 'epoch', 'global_step'].indexOf(name) < 0) {
+      small.push(name);
+    }
+  });
+  if (big.accuracy) {
+    if (big.loss) width = 6;
+    charts.push({
+      config: {lines: big.accuracy},
+      layout: {y: y, h: 2, w: width, x: x},
+    });
+    if (big.loss) x += 6;
+    else y += 1;
+  }
```

Comment:
I'm grouping loss and accuracy to be displayed as two larger charts up top if we have it.  The rest of the metrics are 2x3.